lib/Data/Planner.js

```diff
@@ -52,25 +56,23 @@ export class Planner {
 
         if (typeof requestAnimationFrame === 'function') {
             if (!this._timer) {
-                this._timer = requestAnimationFrame(() => this._flush());
+                this._timer = requestAnimationFrame(() => this.flushAsync());
             }
         } else {
-            this._flush();
+            this.flushAsync();
         }
     }
 
     /**
      * Flush changes
-     *
-     * @private
      */
-    _flush() {
+    flushAsync() {
         let queue = this._commitQueue;
 
         this._timer = null;
         this._commitQueue = [];
 
-        for (var index = 0; index < queue.length; index++) {
+        for (let index = 0; index < queue.length; index++) {
             try {
                 queue[index]();
             } catch(err) {
```

Comment:
I can't understand the necessity of try..catch
What I see is that if callback will throw an error, then Planner will still be consistent